.github/workflows/tf_apply_production.yml

```diff
@@ -16,6 +16,7 @@ env:
   TF_VAR_slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL_PROD }}
   TF_VAR_hashing_peppers: ${{ secrets.PROD_HASHING_PEPPERS }}
   TF_VAR_notify_api_key: ${{ secrets.PROD_NOTIFY_API_KEY }}
+  TF_VAR_shortener_path_length: ${{ secrets.SHORTENER_PATH_LENGTH }}
```

Comment:
Could we just set this in the environment's `terragrunt.hcl` file `input` block?
https://github.com/cds-snc/url-shortener/blob/main/terragrunt/env/staging/api/terragrunt.hcl#L41-L48

Reply:
So how would that work? Would we just have something like this:
```
inputs = {
  ...
  shortener_path_length	   = 8
  }  
```
We could totally do this, I just thought that using a secret might be easier to change since it would not involve terraform code if we ever decide to change the length but I really like your solution since it is much cleaner and overall better.

Reply:
Yup, exactly like that.  The reason I prefer the hardcoding of non-secret values is that it makes it clearer what's being passed into the module.  I have gotten myself into trouble in the past because of how opaque GitHub secrets are and not having an easy way to see the value 😅